### ISSUE TITLE
JHipster Openshift docker image need

### ISSUE THREAD

**Issue by seyfullah** [jhipster/generator-jhipster#8869]
##### **Overview of the feature request**

**jhipster openshift deployment without a docker image.**
I run jhipster openshift command.
Then I get following warnings, I do those.

WARNING! OpenShift configuration generated, but no Jib cache found
If you forgot to generate the Docker image for this application, please run:
To generate the missing Docker image(s), please run:
  ./mvnw package -Pprod jib:dockerBuild in c:\Syf\Hipster\jhipsterdeploymenttest

WARNING! You will need to push your image to a registry. If you have not done so, use the following commands to tag and push the images:
  docker push jhipsterdeploymenttest

I get following error:
The push refers to repository [docker.io/library/jhipsterdeploymenttest]
6844c51dc2f9: Preparing
80ae597b45fc: Preparing
591dc9aeb583: Preparing
bf1c6819c8c0: Waiting
9bca1faaa73e: Waiting
0c3170905795: Waiting
df64d3292fd6: Preparing
denied: requested access to the resource is denied

##### **Motivation for or Use Case**

I cannot advance to next step and deploy this image to my Openshift environment.

##### **Related issues or PR**

No.

-   [X ] Checking this box is mandatory (this is just to show you read everything)

**Comment by seyfullah** [comment]
I cd into src/main/docker folder, I copied war file from target folder to here, and ran the following command.
 docker build -t jhipsterdeploymenttest .

I was leaving folowing question empty:
 What should we use for the base Docker repository name?

Now I answered it as my docker hub account name. Now it says:
WARNING! You will need to push your image to a registry. If you have not done so, use the following commands to tag and push the images:
  docker image tag jhipsterdeploymenttest stikic/jhipsterdeploymenttest
  docker push stikic/jhipsterdeploymenttest

I also ran these commands. Then I ran the following commands as in instructions.
 oc process -f ../ocp/registry/scc-config.yml | oc apply -f -
  oc process -f ../ocp/registry/application-configmap.yml | oc apply -f -
  oc process -f ../ocp/registry/jhipster-registry.yml | oc apply -f -
  oc process -f ../ocp/jhipsterdeploymenttest/jhipsterdeploymenttest-postgresql.yml | oc apply -f -
  oc process -f ../ocp/jhipsterdeploymenttest/jhipsterdeploymenttest-deployment.yml | oc apply -f -

Now when I go to openshift console, There is a pod named as jhipsterdeploymenttest-1-lrnvj .
This has an error like this:

The application will start in 0s...
--
  | Error: Unable to access jarfile //app.war


What is wrong with that?

Thank you.

**Comment by deepu105** [comment]
@srinivasa-vasu do you think there is an issue here?

**Comment by deepu105** [comment]
@seyfullah I don't think you are following what the generator asks you properly. The warning is there to help you. If you want our help you need tl follow our guidelines. As simple as that

**Comment by deepu105** [comment]
This ticket has been closed as the **guidelines are not followed**.

Tickets must follow our [Guidelines](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md), as mentioned in:

1. our [Readme file on the front page of the project](https://github.com/jhipster/generator-jhipster/blob/master/README.md),
2. the ["create a new ticket" page](https://github.com/jhipster/generator-jhipster/issues/new) and
3. our [Help page](http://jhipster.github.io/help/)

We have also created a template on the ["create a new ticket" page](https://github.com/jhipster/generator-jhipster/issues/new) to help you follow those guidelines.

**If this is indeed a JHipster bug**, please open a new issue with proper details or update this issue with all details and request to reopen.
Issues opened without proper details will be closed without explanation.

**If you have a question** please use Stack Overflow, and tag the question with [jhipster](http://stackoverflow.com/questions/tagged/jhipster). This helps the project to keep the issue tracker clean. Also, Stack Overflow will give your question a larger audience:

- This will increase your chances to get an answer
- Answers will be of higher quality, as there is a voting system
- This will also help other users who might have the same issue, as questions are tagged and easily searchable
